src/K1DModel/tendency.jl

```diff
@@ -143,10 +160,12 @@ end
         f_interp.(ρw.components.data.:1),
         dt,
     )
-    # Use the S_Nl tendnecy at cloud base
-    z = CC.Fields.coordinate_field(S_Nl).z # height
-    find_cloud_base(S_Nl, z, cloud_base_S_Nl_and_z)
-    @. S_Nl = ifelse(z == last(cloud_base_S_Nl_and_z), S_Nl, FT(0))
+    if !common_params.local_activation
+        # Use the S_Nl tendnecy at cloud base
+        z = CC.Fields.coordinate_field(S_Nl).z # height
+        find_cloud_base!(S_Nl, z, cloud_base_S_Nl_and_z)
+        @. S_Nl = ifelse(z == last(cloud_base_S_Nl_and_z), S_Nl, FT(0))
```

Comment:
Are we using triangle limiters anywhere in KiD. And if not, should we try to?

Reply:
We use the limiter in tendency computations